earth_enterprise/BUILD_RHEL_CentOS.md

```diff
@@ -115,26 +115,30 @@ prerequisites will be missing and the build will fail.
 Execute:
 
 ```bash
-sudo yum install -y scons perl-Perl4-CoreLibs xorg-x11-server-devel \
-  python-devel perl-Alien-Packages openssl-devel libxml2-devel \
-  libXinerama-devel libXft-devel libXrandr-devel libXcursor-devel gdbm-devel \
-  libmng-devel libcap-devel libpng12-devel libXmu-devel freeglut-devel \
-  zlib-devel libX11-devel bison-devel openjpeg-devel openjpeg2-devel \
-  geos-devel proj-devel ogdi-devel giflib-devel xerces-c xerces-c-devel \
-  cmake GitPython rpm-build rpmrebuild rsync
+sudo yum install -y \
+  bison-devel cmake daemonize freeglut-devel \
+  gdbm-devel geos-devel giflib-devel GitPython \
+  libcap-devel libmng-devel libpng12-devel libX11-devel libXcursor-devel \
+    libXft-devel libXinerama-devel libxml2-devel libXmu-devel libXrandr-devel \
```

Comment:
Minor: inconsistent indentation

Reply:
There's no clear reason (at least to me) why this one line should be a sub-level when nothing else is.

Reply:
They're ordered alphabetically.  And, this continues the same letter, rather than starting a new one.